src/appendices/site-user-config-ref.rst

```diff
@@ -195,7 +195,7 @@ URL of the cylc internet homepage, with links to documentation for the
 latest official release.
 
 - *type*: string
-- *default*: http://cylc.github.com/cylc/
+- *default*: https://github.com/cylc/cylc-flow
```

Comment:
Why?

Reply:
Since the description for the setting, as per the line above, is "URL of the cylc internet homepage" which is as suggested, whereas the current URL points to the GitHub repo?

Reply:
Derp.